apps/examples/testcase/ta_tc/systemio/itc/itc_i2c.c

```diff
@@ -115,15 +116,26 @@ void itc_iotbus_i2c_set_address_p(void)
 void itc_iotbus_i2c_write_read_p(void)
 {
 	int ret = IOTBUS_ERROR_NONE;
-	uint8_t szCmd[2] = { 0x01, 0x02 };
-	uint8_t szReadBuf[10];
+	uint8_t write_cmd[2] = { 0x1, 0x2};
```

Comment:
You didn't modify here.

Reply:
Do you want modification like below or else , please suggest:
 uint8_t write_cmd[2] = { 1, 2};